### ISSUE TITLE
Question about train LeViT-256

### ISSUE THREAD

**Issue by mingqiJ** [facebookresearch/LeViT#26]
Hi,

I am training LeViT-256. I use this command: 
`python3 -m torch.distributed.launch --nproc_per_node=4 --use_env main.py --model LeViT_256 --data-path /scratch/pytorch-image-models-master-mingqi/imagenet --output_dir /output --seed 0 --batch-size 512`. 
But it takes this error.
![1931661622197_ pic](https://user-images.githubusercontent.com/98499594/187041884-22acc389-1c75-470e-9545-a40dff753d40.jpg)

It seems like package version problem? or python version problem? Is there anyone could help me solve this problem?

Thanks

**Comment by btgraham** [comment]
It seems the external function timm.create_model has added an extra argument.
I have updated the repo -- please git pull and try again.
